### ISSUE TITLE
concerning the results in Table 4

### ISSUE THREAD

**Issue by Daniellli** [SimingYan/IAE#12]
Hi, thanks for your such great work.
i am trying to reproducing the results in Table 4 with the [codebase](https://github.com/SimingYan/dgcnn.pytorch) and   [pretrained model ](https://drive.google.com/drive/folders/1-5yVQBQPLoEZjI4oBz2pGh67vxveBq2M?usp=sharing).  The finetune script i used is : 

```
python main_semseg.py \
--exp_name=semseg_66 --test_area=6 \
--batch_size=16 \
--restore_path=pretrained_models/dgcnn_semseg.pt \
```

and the evaluation script: 

```

python main_semseg.py \
--exp_name=semseg_eval --test_area=all \
--eval=True --model_root=outputs/semseg_66/models/model_6.t7 
```

However, i the results higher than the one listed in Table4 :  

<!DOCTYPE html><figure class="md-table-fig" cid="n541" mdtype="table" style="box-sizing: border-box; margin: 1.2em 0px; overflow-x: auto; max-width: calc(100% + 16px); padding: 0px; cursor: default; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;Open Sans&quot;, &quot;Clear Sans&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;Segoe UI Emoji&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none;">

  | OA | mIoU
-- | -- | --
finetue with dgcnn_semseg.pt | 95.25 | 86.1

</figure><p cid="n470" mdtype="paragraph" class="md-end-block md-p" style="box-sizing: border-box; line-height: inherit; orphans: 4; margin: 0.8em 0px; white-space: pre-wrap; position: relative; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;Open Sans&quot;, &quot;Clear Sans&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;Segoe UI Emoji&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; text-align: start; text-indent: 0px; text-transform: none; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none;"></p><p cid="n499" mdtype="paragraph" class="md-end-block md-p" style="box-sizing: border-box; line-height: inherit; orphans: 4; margin: 0.8em 0px; white-space: pre-wrap; position: relative; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;Open Sans&quot;, &quot;Clear Sans&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;Segoe UI Emoji&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; text-align: start; text-indent: 0px; text-transform: none; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none;"></p><p cid="n501" mdtype="paragraph" class="md-end-block md-p" style="box-sizing: border-box; line-height: inherit; orphans: 4; margin: 0.8em 0px; white-space: pre-wrap; position: relative; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;Open Sans&quot;, &quot;Clear Sans&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;Segoe UI Emoji&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; text-align: start; text-indent: 0px; text-transform: none; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none;"></p><br class="Apple-interchange-newline">

Is there anything wrong? why is it so high? 

appreciate for your help .

**Comment by SimingYan** [comment]
Hi,

I am not totally sure but I think the reason might be this: if you want to evaluate on all the test area, you should train 6 different models. Please take a look at how to cross-validation on S3DIS.

**Comment by Daniellli** [comment]
Hi, thank you for your quick response.  
I am not familiar with point cloud semantic segmantation and corespoding dataset.  why i do so refer to [ your released traninig log](https://drive.google.com/drive/folders/1hNitNNnpAuO7vniW0mCFOrFj8nkXQ6Ne) which  guide me to evaluate on all area  can get the results of DGCNN shown in table 4.  

now, i know how to repreduce the result: training 6 model in 6 different training setting then calculating the  overall accuracy and mIoU.  Is it correct? 

appreciate for your help and such novel work.

**Comment by SimingYan** [comment]
- now, i know how to repreduce the result: training 6 model in 6 different training setting then calculating the overall accuracy and mIoU. Is it correct?
- Yeah. It's correct.
